airbyte-ci/connectors/connectors_qa/src/connectors_qa/checks/documentation/templates/documentation_headers_check_description.md

```diff
@@ -0,0 +1,17 @@
+The user facing connector documentation should follow the guidelines defined in the [documentation standards](https://hackmd.io/Bz75cgATSbm7DjrAqgl4rw).
```

Comment:
Please do not reference this `hackmd.io`. The work you've done should actually be enough to document the expected structure of documentation right?